docsets/Bokeh/docset.json

```diff
@@ -1,13 +1,20 @@
 {
-  "name": "Bokeh",
-  "version": "0.12.3",
-  "archive": "Bokeh.tgz",
-  "author": {
-    "name": "Joseph Crail",
-    "link": "https://github.com/jbcrail"
-  },
-  "aliases": ["Python interactive visualization library",
-              "Bokeh Scientific plots",
-              "web browser visualization"
-              ]
-}
+    "name": "Bokeh",
+    "version": "2.0.1",
+    "archive": "Bokeh.tgz",
+    "author": {
+      "name": "Angelo Varlotta",
+      "link": "https://github.com/capac"
+    },
+    "aliases": ["Python interactive visualization library",
+                "Bokeh Scientific plots",
+                "web browser visualization",
+                "bokeh"],
+
+    "specific_versions": [
+        {
+            "version": "0.12.3",
+            "archive": "versions/0.12.3/Bokeh.tgz",
```

Comment:
You need to actually add this archive. Moving the txt file doesn't have any effect.